oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -148,17 +148,17 @@ public class VersionGarbageCollector {
      */
     static final String SETTINGS_COLLECTION_DETAILED_GC_DRY_RUN_DOCUMENT_ID_PROP = "detailedGCDryRunId";
 
-    static enum RDGCType {
+    enum RDGCType {
         NO_OLD_PROP_REV_GC,
         KEEP_ONE_FULL_MODE,
         KEEP_ONE_CLEANUP_USER_PROPERTIES_ONLY_MODE,
         OLDER_THAN_24H_AND_BETWEEN_CHECKPOINTS_MODE
     }
 
-    private static RDGCType revisionDetailedGcType = RDGCType.NO_OLD_PROP_REV_GC;
+    private static RDGCType REVISION_DETAILED_GC_TYPE = RDGCType.NO_OLD_PROP_REV_GC;
```

Comment:
Changing this would fail `VersionGarbageCollectorIT` as it for example overwrites the value in [setUp()](https://github.com/apache/jackrabbit-oak/blob/21e8b289447790b69c78291335983b19c9471a7c/oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollectorIT.java#L229)

Reply:
as it is non final, does it need to be changed to capitals?

Reply:
ah, good point. Will revert this as well.